.github/workflows/branches.yml

```diff
@@ -29,13 +29,6 @@ jobs:
         env:
           DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
         run: docker login -p "${DOCKER_TOKEN}" -u etnaagent
-      - name: Download all images
```

Comment:
In the past, image dependencies (things that build off other images) were not explicit.  Instead, all images under docker were built first, then each application image was built next.  This was simple, but the lack of clear dependencies meant that during build in CI, it wasn't immediately clear which images needed to be downloaded, and thus that work was serialized quite a bit, resulting in slow processes.

This script used to "en-masse" download all base images to prepare for the work.

It's not longer necessary because image dependencies are determined, and requisite images are downloaded as they are needed.